apps/entity-service/src/entities/processors/entity-processor.ts

```diff
@@ -169,6 +176,71 @@ export abstract class EntityProcessor<
   loadAssociationData(ids: number[]): Promise<Record<number, object>> {
     return Promise.resolve({});
   }
+
+  private async findReportsByUuids<T extends ReportModel>(
+    modelClass: ModelCtor<T>,
+    uuids: string[] | undefined
+  ): Promise<T[]> {
+    if (uuids == undefined) return [];
```

Comment:
```suggestion
  private async findReportsByUuids<T extends ReportModel>(
    modelClass: ModelCtor<T>,
    uuids: string[]
  ): Promise<T[]> {
    if (uuids == null) return [];
```

I think this is better in case the value becomes null or undefined, and we force to receive a value